Sources/Subs-Post.php

```diff
@@ -1316,12 +1316,22 @@ function smtp_mail($mail_to_array, $subject, $message, $headers)
 	if (!server_parse(null, $socket, '220'))
 		return false;
 
+	// Try and determine the servers name, fall back to the mail servers if not found
+	$helo		= FALSE;
```

Comment:
Just a couple of cosmetic things, SMF doesn't use uppercase for boolean values and theres 2 tabs before the = sign, we usually use a single space.